ts/src/core/privacy.ts

```diff
@@ -1,16 +1,16 @@
 import {
-  Viewer,
-  ID,
+  Allow,
+  Context,
+  Deny,
   Ent,
+  ID,
```

Comment:
what rule do you have that automatically reorders these?

Reply:
VSCode settings:
```json
"editor.codeActionsOnSave": {
  "source.organizeImports": true
},
```